pkg/healthcheck/healthgo_test.go

```diff
@@ -126,7 +129,7 @@ func getHealthCheckerWithPassingHTTPCheck(t *testing.T) healthcheck.HealthChecke
 	t.Helper()
 	a := assertions.New(t)
 	hc := getDefaultHealthChecker(t)
-	err := hc.AddHTTPCheck("test-http-check", "http://localhost:3324")
+	err := hc.AddHTTPCheck("test-http-check", "http://example.com")
```

Comment:
Do we really need this external dependency?

Reply:
I felt that starting an extra server just to have something to hit on was quite overkill. If we see flakes on this domain we'll change it.